docs/jotai/api/utils.mdx

```diff
@@ -222,6 +222,33 @@ import deepEqual from 'fast-deep-equal'
 const fooFamily = atomFamily((param) => atom(param), deepEqual)
 ```
 
+### TypeScript
+
+Types are defined like this:
+
+```ts
+atomFamily<Param, Value, Update>(initializeAtom: (param: Param) => WritableAtom<Value, Update>, areEqual?: (a: Param, b: Param) => boolean)
+atomFamily<Param, Value>(initializeAtom: (param: Param) => Atom<Value>, areEqual?: (a: Param, b: Param) => boolean)
+```
+
+It's generally recommended to infer types from initializeAtom.
```

Comment:
"The atom family types will be inferred from initializeAtom."  

I think we can mention this part first and then talk about generics.